pages/report.tsx

```diff
@@ -127,6 +177,23 @@ export default function Report() {
                     <SharkWasReleasedCheckboxField name="was_released" />
                   </Box>
 
+                  <Box>
+                    <FormControl mb={5}>
+                      <FormLabel>We accept up to 4 images under 5MB</FormLabel>
```

Comment:
Two things:
1. We aren't actually enforcing the "number of files" limit
2. From a UX perspective, the label should be a prompt that motivates the user to take action. This is just a declaration of what we accept, which is more appropriate for the hint/helper text instead. A more direct instruction for the user is appropriate here- something like, "Upload images of the sharks" or "Do you have pictures of the sharks?"